### ISSUE TITLE
First test

### ISSUE THREAD

**Issue by Walther** [Walther/clovers#20]
Implement the first test for the project.

Criteria:

- renders a small, fast-to-render scene
- the file is saved successfully
- can be run on three major operating systems (Windows, MacOS, Linux)
- is run on CI for each PR

**Comment by Walther** [comment]
Depends on #12 due to https://doc.rust-lang.org/book/ch11-03-test-organization.html#integration-tests-for-binary-crates

**Comment by Walther** [comment]
Closing in favor of other issues. Some small unit tests have already been added.
